simulator/core/t/unit_test.cpp

```diff
@@ -9,7 +9,7 @@
 #include "../perf_sim.h"
 
 static const std::string valid_elf_file = TEST_PATH;
-static const int64 num_steps = 2039;
+static const int64 num_steps = 2041;
```

Comment:
It does not seem right. Have you tried to run the trace `tt.core.out`?

Reply:
Yes, I tried and all was OK but apparently there should be a number larger

Reply:
OK. Unfortunately there is no other way to find it than substitution.

Note that tt.corre.s resides in the submodule, so you have to make a pull request there as well.